### PR TITLE
Adding note sharing and Modification of emojis

### PR DESCRIPTION
**Pull request by godetremy**
## Pull request checklist

Please check if your PR fulfills the following requirements:

- [X] You can **build successfully** the whole project with your changes locally
- [X] You use the coding conventions and the **naming conventions** of the project
- [X] You use **tabs** for indentation
- [X] You make sure that your PR is **not a duplicate**
- [X] This PR is **ready** to be reviewed and merged
- [X] This PR merges into the **`development`** branch or in the specific branch of the feature you want to merge into
- [X] There is no **`TODO`** in the code
- [X] There is no spelling or grammatical errors in the code
- [X] Details of the issue or feature are documented below
- [X] This PR is **not** a **breaking change** (e.g. changes that would cause existing functionality to change)

### Adding note sharing

Following the addition of the "note.html" page to the getpapillon.xyz website, the next step was to modify the note sharing function in order to retrieve the data, encode it, and then create a shareable URL.

### Modification of emojis

In view of the [bug-report Discord](https://discord.com/channels/1014931881906675712/1181478842855071897), I added the following emojis:

SI: 💻

SNT: 💻

EPS: 🏀

Latin: 🏛️

EMC: 🤝

SES: 💰

### Note

Before merging this PR, you will need the approval of at least one of the following people:

- @ecnivtwelve
- @lucas-luchack

So be patient ;)

### CODE REVIEW

**Review by ecnivtwelve — APPROVED** [review]
Ca me paraît bon, je merge dans l'aprém

**Comment by LeGeek01** [comment]
2 semaines plus tard mdr

**Comment by godetremy** [comment]
Les après-midi sont long chez Vincent... 💀

**Comment by LeGeek01** [comment]
@ecnivtwelve alors ? mdr

**Comment by ecnivtwelve** [comment]
long l'aprèm

**Comment by godetremy** [comment]
2 semaine que j'attend et Vincent OSE MERGE QUELQU'UN D'AUTRE ! 💀

**Comment by ecnivtwelve** [comment]
> 2 semaine que j'attend et Vincent OSE MERGE QUELQU'UN D'AUTRE ! 💀

il a la flemme ouu ???

**Comment by LeGeek01** [comment]
@ecnivtwelve il est lent ce lait

**Comment by LeGeek01** [comment]
ah bah voilà maintenant y'a des conflits ptdr
image

File: views/Grades/GradeView.js [hunk 1]
```diff
@@ -155,16 +159,48 @@ function calculateExactGrades(grades, isClass) {
...
 
-  function shareGrade() {
-    Alert.alert(
-      'Partager la note',
-      "Le partage de la note n'est pas encore disponible.",
-      [{ text: 'OK' }]
-    );
+  async function shareGrade(grade, color) {
+    const baseURL = "https://getpapillon.xyz/note"
+    const comment = ["Une franche réussite","Qu'elle succès","Absolument fantastique","Épatant","Un travail exceptionnel","Remarquable","Impressionnant","Merveilleux","À couper le souffle","Brillant","Magique","Extraordinaire","Stupéfiant","Inoubliable","Exceptionnellement bien fait","Éblouissant","Génial","Sublime","Incroyable","Époustouflant","Travail consciencieux","Bonne performance","En progrès constant","Très réussi","Mérite une mention","Bien joué !","Excellent travail !","Bravo, très bien !","Très satisfaisant","Félicitations !","Travail exceptionnel",]
```
Comment (Yaya-Cout) [inline views/Grades/GradeView.js:174]: Ce n'est pas plutôt "Quel succès" ?
```suggestion
    const comment = ["Une franche réussite","Quel succès","Absolument fantastique","Épatant","Un travail exceptionnel","Remarquable","Impressionnant","Merveilleux","À couper le souffle","Brillant","Magique","Extraordinaire","Stupéfiant","Inoubliable","Exceptionnellement bien fait","Éblouissant","Génial","Sublime","Incroyable","Époustouflant","Travail consciencieux","Bonne performance","En progrès constant","Très réussi","Mérite une mention","Bien joué !","Excellent travail !","Bravo, très bien !","Très satisfaisant","Félicitations !","Travail exceptionnel",]
```

Comment (LeGeek01) [reply views/Grades/GradeView.js:174]: si ptdr

Comment (godetremy) [reply views/Grades/GradeView.js]: ChatGPT... 💀

**Review by LeGeek01 — APPROVED** [review]
c'est mieux

**Pull request merged**